src/tests/ut/account/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+add_executable(account-ut account_test.cpp)
```

Comment:
Is this CMake used to have a separated target?

Reply:
It willl create additional test target with ut-only. I think, that this might be a good idea for the time being. Helps track memory issues.

We could make additonal targets for all ut-s and track which ones still have memory issues.

All in all, I think the ultimate step might be to resign from single test target and test only separate targets with ctest with one command. But we must first incorporate ctests into to our CI workflow.